packages/stellar-axelar-std-derive/src/storage.rs

```diff
@@ -261,19 +296,30 @@ fn fields_data(fields: &Fields) -> (Vec<&Option<Ident>>, Vec<&Type>) {
             let types = fields.named.iter().map(|f| &f.ty).collect();
             (names, types)
         }
-        _ => panic!("Only unit variants or named fields are supported in storage enums."),
+        _ => panic!("only unit variants or named fields are supported in storage enums"),
     }
 }
 
-fn fn_names(variant: &Variant) -> (Ident, Ident, Ident) {
-    (
-        format_ident!("{}", variant.ident.to_string().to_snake_case()),
-        format_ident!("set_{}", variant.ident.to_string().to_snake_case()),
-        format_ident!("remove_{}", variant.ident.to_string().to_snake_case()),
-    )
+fn fn_names(variant: &Variant, status: bool) -> (Ident, Ident, Ident) {
+    if status {
+        (
+            format_ident!("is_{}", variant.ident.to_string().to_snake_case()),
```

Comment:
define a variable for `variant.ident.to_string().to_snake_case())` once and use everywhere